.circleci/config.yml

```diff
@@ -20,10 +20,12 @@ jobs:
       - run: git fetch origin
       - run: git config --global user.name LinuxbrewTestBot
       - run: git config --global user.email testbot@linuxbrew.sh
-      - run: chmod 0644 *.rb
+      - run: chmod 0644 Formula/*.rb
       - run: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
       - run: cp -a . /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew/$CIRCLE_PROJECT_REPONAME
-      - run: |
+      - run:
+         no_output_timeout: 30m
+         command: |
```

Comment:
The indentation here is odd. There's three spaces of indent between run and command, and only one space between command and the next chunk.